pkg/ocicni/ocicni.go

```diff
@@ -163,6 +205,15 @@ func (plugin *cniNetworkPlugin) setDefaultNetwork(n *cniNetwork) {
 	plugin.defaultNetwork = n
 }
 
+func (plugin *cniNetworkPlugin) terminateMonitorNetDir(err error) {
+	plugin.Lock()
+	defer plugin.Unlock()
+	if !plugin.monitorNetDirDone {
```

Comment:
@rajatchopra We don't need it, this is a copy/paste from my previous PR. This is fixed now.